### ISSUE TITLE
PAUL & DESTA

### ISSUE THREAD

**Issue by canon20paul** [canon20paul/Bootstrap-My-Portfolio#2]

**Comment by dlwub** [comment]
I don't think it is a good habit to leave the alternative text (alt) empty. I observed that in the code https://github.com/canon20paul/Bootstrap-My-Portfolio/blob/Feature/index.html#L123-L289. Alt fields help people with vision impairment to recognize what is displayed in the images.
